tslint.json

```diff
@@ -27,6 +27,16 @@
 		"readonly-array": true,
 		"readonly-keyword": [true, "ignore-class"],
 		"strict-boolean-expressions": false,
-		"strict-type-predicates": false
+		"strict-type-predicates": false,
+		// Better to have functionally similar items be close together.
+		"ordered-imports": false,
+		// We need private class properties to start with an underscore.
+		"variable-name": [true, "allow-leading-underscore"],
+		// Better to have functionally similar items be close together.
+		"member-ordering": false,
+		// This rule can sometimes encourage bad practices; better leave this to the PR process.
+		"prefer-function-over-method": false,
+		// The shorthand arrow notation is often more difficult and error-prone to refactor.
+		"arrow-return-shorthand": true
```

Comment:
In general, I think this is just which coding style we choose. Once we get use to it, I dont think it's so confusing.
Also, this rule is applied from `lisk-template` so I would prefer to keep it as all the project suppose to be following.
For example
```
{
// ...
  (peerObject: ProtocolPeerInfo) => new Peer({
      ipAddress: peerObject.ip,
      wsPort: peerObject.wsPort, // TODO ASAP: Add more properties
      nodeStatus: this._nodeStatus,
    });
};
```
or if it's just returning the object
```
{
// ...
  (peerObject: ProtocolPeerInfo) => ({
      ipAddress: peerObject.ip,
      wsPort: peerObject.wsPort, // TODO ASAP: Add more properties
      nodeStatus: this._nodeStatus,
    });
}
```

Reply:
My argument is that this would be better judged on a case-by-case basis than with a hard rule. I don't think that rule adds any value at all.

Reply:
I think this is more of just for consistency. Rather than case-by-case, I would like to keep it consistent

Reply:
https://github.com/airbnb/javascript#arrows--implicit-return
this is the related eslint enabled in lisk-template